### ISSUE TITLE
Change size of the view that contain the chart

### ISSUE THREAD

**Issue by mlorenzeCDS** [danielgindi/ios-charts#440]
How is the best way to manage the redraw of the chart when the size of the View that contain the chart is modify (big view to small and the other way around?

The chart size is modify too, that the problem with the transition of the change of the view (stuttering).

Thanks for your help!

**Comment by danielgindi** [comment]
Can you record a demo to let us see what you are experiencing? Basically the chart view is handling resize by itself, you don't have to do nothing...

**Comment by danielgindi** [comment]
Closed for no response.

**Comment by ledanh57** [comment]
Hi there,

I guess I'm having the same problem here and yet to figure out. At first I init the bar chart for all days in a certain month. It works just great but if I switch to year by changing the count of BarChartDataSet to 12 (12 months), then I ran into problem where some months displayed twice or even triple.

I found out that if I init for the year first, it would be fine until I switch to month and then back. Here comes some screenshot:

1. If I init the year first, it should be fine:
<img width="283" alt="Screen Shot 2019-04-22 at 6 40 28 PM" src="https://user-images.githubusercontent.com/47553533/56498927-401c2c80-652e-11e9-8441-934061defa41.png">

2. In case I init month first, then when it comes to year it'd be like this:
<img width="290" alt="Screen Shot 2019-04-22 at 6 40 46 PM" src="https://user-images.githubusercontent.com/47553533/56498933-44484a00-652e-11e9-84fd-4905bc8e55e2.png">

Should there be any method to redraw the view when count changed?
